index.html

```diff
@@ -8,37 +8,53 @@
     />
     <title>Product Preview</title>
     <link rel="stylesheet" type="text/css" href="styles.css" />
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Fraunces:ital,opsz,wght@0,9..144,100..900;1,9..144,100..900&family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Fraunces:ital,opsz,wght@0,9..144,100..900;1,9..144,100..900&family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <div class="wrapper">
-      <div class="flexbox-container">
+    <main class="main-content-wrapper">
+      <div class="card-component-container">
         <picture>
-          <source  srcset="/product-preview-card-component-main/images/image-product-mobile.jpg" media="(max-width: 600px)" />
-        <img
-          class="product-image"
-          src="/product-preview-card-component-main/images/image-product-desktop.jpg"
-          alt="An image of the product"
-        />
-      </picture>
-        <div class="product-description">
-          <h4>PERFUME</h4>
-          <h2>Gabrielle Essence Eau De Parfum</h2>
-          <p>
-            A floral, solar and voluptuous interpretation composed by Olivier Polge, 
-  Perfumer-Creator for the House of CHANEL.
+          <source
+            srcset="
+              /product-preview-card-component-main/images/image-product-mobile.jpg
+            "
+            media="(max-width: 600px)"
+          />
+          <img
+            class="product-image"
+            src="/product-preview-card-component-main/images/image-product-desktop.jpg"
+            alt="An image of the product"
+          />
+        </picture>
+        <article class="product-description-box">
+          <h2 class="product-type">perfume</h2>
+          <h1 class="product-name">
+            Gabrielle Essence Eau De Parfum
+          </h1>
+          <p class="product-description">
+            A floral, solar and voluptuous interpretation composed by
+            Olivier Polge, Perfumer-Creator for the House of CHANEL.
           </p>
-          <span>$149.99</span> <del>$169.99</del>
-          <button>
+          <ins class="new-price">$149.99</ins>
+          <del class="old-price">$169.99</del>
+          <button class="btn">
```

Comment:
w przypadku takiego konkretnego przycisku jak 'add to cart' warto dodać to w nazwie klasy (np add-to-cart-button) - z zasady nie powinno się łączyć nazwy klasy z treścią przycisku (bo w razie potrzeby trzeba zmieniać w conajmniej dwóch miejscach), ale tutaj mamy treść przycisku, która wskazuje też na jego funkcję, będzie to 'add to cart' niezależnie od tekstu w środku. Jeśli chodzi o 'btn' to jest to spoko klasa dla typowego przycisku gdzieś na stronie, takiego bez wyróżniającego się stylu czy wykonywanego zadania